src/App.js

```diff
@@ -1,7 +1,19 @@
 import "./App.css";
 
 function App() {
-  return <div className="App">Hello World</div>;
+  return (
+    <div className="App">
+      <header className="App-header">My cakes</header>
+      <main className="App-main">Content</main>
+      <footer className="App-footer">
```

Comment:
Namir once mentioned, we dont need to use the "App", since a react thing.. so i would just go with header, main and footer

Reply:
Yes, good point.